### ISSUE TITLE
Error: Cannot access static:: when no class scope is active

### ISSUE THREAD

**Issue by uok** [matthiasmullie/minify#208]
In minify v1.3.53 I get this error:
"Fatal error: Cannot access static:: when no class scope is active in /minify/src/JS.php on line 233"

Any idea what this means? Thanks!

**Comment by Tabrisrp** [comment]
I have the same issue on PHP 5.3.29.

No issue on PHP 5.4.

Backtrace
```
MatthiasMullie\Minify\Minify->minify() /app/minify.php:247
MatthiasMullie\Minify\JS->execute() /app/vendor/matthiasmullie/minify/src/Minify.php:102
MatthiasMullie\Minify\Minify->replace() /app/vendor/matthiasmullie/minify/src/JS.php:174
MatthiasMullie\Minify\Minify->replacePattern() /app/vendor/matthiasmullie/minify/src/Minify.php:271
preg_replace_callback() /app/vendor/matthiasmullie/minify/src/Minify.php:309
MatthiasMullie\Minify\{closure}() /app/vendor/matthiasmullie/minify/src/Minify.php:309
```

**Comment by matthiasmullie** [comment]
Thanks for the report, @uok and @Tabrisrp! I believe this is now fixed - can you confirm?

**Comment by uok** [comment]
Thanks for fixing, the error has now changed to 
"Fatal error: Call to protected method MatthiasMullie\Minify\Minify::extractStrings() from context '' in /minify/src/JS.php on line 243"

**Comment by matthiasmullie** [comment]
Hey @uok 
Thanks for testing! Any chance you could try again with current master? :)

**Comment by uok** [comment]
Thanks! Current master gives "Fatal error: Using $this when not in object context in /minify/src/JS.php on line 250"

**Comment by matthiasmullie** [comment]
Ugh - I seem to be slowly forgetting the kind of workarounds that were required for PHP5.3 support :D
I just added another commit - can you try again? Thanks!

**Comment by uok** [comment]
No worries ;-)
latest master shows "Cannot access protected property MatthiasMullie\Minify\JS::$nestedExtracted in /minify/src/JS.php on line 250"

**Comment by matthiasmullie** [comment]
Well well - we're closing in... Can you try again please? :)

**Comment by uok** [comment]
That did the trick 👍 
Thanks for taking the time to fix this for PHP 5.3

**Comment by matthiasmullie** [comment]
Alright! Thanks for bearing with me while we got through this!
